### PR TITLE
[WIP] Add IGListKit Swift helpers

### PR DESCRIPTION
**Pull request by rnystrom**
Messing around w/ an idea I had about how to reduce the amount of `guard` required to work w/ IGLK when stuff like the `collectionContext` should always exist.

### CODE REVIEW

File: Classes/Section Controllers/IGListSectionController+Utils.swift [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+extension ListSectionController {
+
+    var util_context: ListCollectionContext {
+        guard let context = collectionContext else {
+            fatalError("Missing context")
+        }
+        return context
+    }
+
+    func util_dequeueCell<T: AnyObject>(index: Int) -> T {
```
Comment (BasThomas) [inline Classes/Section Controllers/IGListSectionController+Utils.swift]: Maybe keep the naming consistent here (`dequeueReusableCell`) for easier lookup via autocompletion? Also not sold on the `util_`, think it's pretty hard to discover atm 😕

**Comment by rnystrom** [comment]
ignore the requests, testing PR request feature

**Comment by Sherlouk** [comment]
Interesting how removing the request for review from Bas also removed his actual review from the sidebar 👀

**Comment by rnystrom** [comment]
That's... kind of weird...

**Comment by rnystrom** [comment]
I'll come back to this sometime

**Pull request closed without merging**